packages/samples/headless-react/src/pages/SearchPage.tsx

```diff
@@ -356,6 +356,34 @@ export class SearchPage extends Component {
     this.unsubscribeExecuteTrigger();
   }
 
+  private get messageExpression() {
+    return buildQueryExpression({
+      operator: 'and',
```

Comment:
We could make `operator` optional with `and` as the default. It was the value used in all the samples. It's also technically not necessary to specify if only adding one expression, so it would reduce the verbosity in this case,

Flip argument is making it required makes it clear how expressions are going to be concatenated, and helps with discoverability 🤷

Reply:
I would :+1: using `and` as by default

Reply:
I think it's fine to use `and` as the default. My only worry is that, since this is a builder, some people may be looking for a `withOperator` method instead of a parameter.

Reply:
True, a method would be more aligned with the builder pattern, will adjust to that